docs/definition.md

```diff
@@ -7,42 +7,55 @@ A **Fern API Definition** is a set of YAML files that describe your API. Each fi
 - **[Errors](errors.md)**: error handling
 - **[IDs](ids.md)**: unique identifiers
 - **[Imports](imports.md)**: share types, errors, and ids across YAML files
+- **[Docs](docs.md)**: add descriptions throughout your definition
+
+## Explore Fern API Definitions on Github
+
+- [Connect Earth](https://github.com/fern-api/fern-connect-earth/blob/main/api/src/charts.yml) helps embed sustainability into financial products.
+
+- [Telematica](https://github.com/fern-api/fern-telematica/blob/main/api/src/vehicleData.yml) is a unified API for electric vehicles.
 
 ## An example of a Fern API Definition
 
 ```yml
-# This is a sample IMDb API to get you more familiar with defining APIs in Fern.
+# A sample API for IMDb (International Movie Database)
```

Comment:
I think we can get rid of `A sample API for IMDb (International Movie Database)`